README.rst

```diff
@@ -540,10 +541,60 @@ Resources and Links
 For more details, see:
 
   - `Legacy PyMAPDL Reader Documentation <https://readerdocs.pyansys.com/>`_
-  - `Legacy PyMAPDL Reader PyPi <https://pypi.org/project/ansys-mapdl-reader/>`_
+  - `Legacy PyMAPDL Reader PyPI <https://pypi.org/project/ansys-mapdl-reader/>`_
   - `Legacy PyMAPDL Reader GitHub <https://github.com/pyansys/pymapdl-reader>`_
 
 
+Granta MI BoM Analytics
+-----------------------
+The Granta MI Restricted Substances solution includes BoM Analytic Services,
+which provides a REST API to allow external applications and tools to determine
+the compliance of materials and products against various legislations. This
+package provides a Pythonic interface to the BoM Analytic Services API.
+
+Installation
+~~~~~~~~~~~~
+Install this package with:
+
+.. code::
+
+   pip install ansys-grantami-bomanalytics
+
+Usage
+~~~~~
+Here's a brief example of how this package works:
+
+.. code:: python
+
+    # Connect and query the Granta service.
+
+    >>> from pprint import pprint
+    >>> from ansys.grantami.bomanalytics import Connection, queries
+    >>> cxn = Connection(servicelayer_url='http://localhost/mi_servicelayer').with_autologon().connect()
```

Comment:
Do we have an agreed style for remote resources? We should probably agree on HTTPS at least, if not a common URL?

Reply:
In the bomanalytics examples we use `http://my_grantami_server/mi_servicelayer` as the placeholder URL. I think as a first step we should at least be self-consistent within the package.

```suggestion
    >>> cxn = Connection(servicelayer_url='http://my_grantami_server/mi_servicelayer').with_autologon().connect()
```

As for HTTPS, I don't know. We should encourage people to use HTTPS by default, but I think ultimately the majority do not.

Reply:
I have created an issue https://github.com/pyansys/pyansys_landing/issues/17. It's not strictly speaking purely a landing page issue, but it does manifest here because everything is concatenated together.